src/components/Capture/Capture.tsx

```diff
@@ -1,10 +1,10 @@
-import React, { useEffect, useState } from 'react';
+import { useEffect, useState } from 'react';
```

Comment:
We need to remember later, when we make react-16 release to add this import on branch with 16

Reply:
And in other places where this import is removed